tests/test-post-connection-queries.php

```diff
@@ -332,4 +332,198 @@ public function testPageWithChildren() {
+			'categoryIn' => [4, 5, 6],
+		];
+
+		$app_context = new \WPGraphQL\AppContext();
+		
+		$app_info = new \GraphQL\Type\Definition\ResolveInfo( array() );
+
+		$actual = \WPGraphQL\Type\PostObject\Connection\PostObjectConnectionResolver::sanitize_input_fields( $mock_args, null, [], $app_context, $app_info );
+
+		/**
+		 * Make sure the returned values are equal to mock args
+		 */
+		$this->assertEquals( 1, $actual['cat'] );
+		$this->assertEquals( 'testCategory', $actual['category_name'] );
+		$this->assertEquals( [4, 5, 6], $actual['category__in'] );
+
+		/**
+		 * Make sure the query didn't return these array values
+		 */
+		$this->assertArrayNotHasKey( 'categoryId', $actual );
+		$this->assertArrayNotHasKey( 'categoryName', $actual );
+		$this->assertArrayNotHasKey( 'categoryIn', $actual );
+	}
+
+	public function testSanitizeInputFieldsTagArgs() {
+		$mock_args = [
+			'tagId' => 1,
+			'tagIds' => [1, 2, 3],
+			'tagSlugAnd' => [4, 5, 6],
+			'tagSlugIn' => [6, 7, 8],
+		];
+
+		$app_context = new \WPGraphQL\AppContext();
+		
+		$app_info = new \GraphQL\Type\Definition\ResolveInfo( array() );
+
+		$actual = \WPGraphQL\Type\PostObject\Connection\PostObjectConnectionResolver::sanitize_input_fields( $mock_args, null, [], $app_context, $app_info );
+
+		/**
+		 * Make sure the returned values are equal to mock args
+		 */
+		$this->assertEquals( 1, $actual['tag_id'] );
+		$this->assertEquals( [1, 2, 3], $actual['tag__and'] );
+		$this->assertEquals( [4, 5, 6], $actual['tag_slug__and'] );
+		$this->assertEquals( [6, 7, 8], $actual['tag_slug__in'] );
+
+		/**
+		 * Make sure the query didn't return these array values
+		 */
+		$this->assertArrayNotHasKey( 'tagId', $actual );
+		$this->assertArrayNotHasKey( 'tagIds', $actual );
+		$this->assertArrayNotHasKey( 'tagSlugAnd', $actual );
+		$this->assertArrayNotHasKey( 'tagSlugIn', $actual );
+	}
+
+	public function testSanitizeInputFieldsSearchArgs() {
+		$mock_args = [
+			'search' => 'testSearchString',
+			'id' => 1,
+		];
+
+		$app_context = new \WPGraphQL\AppContext();
+		
+		$app_info = new \GraphQL\Type\Definition\ResolveInfo( array() );
+
+		$actual = \WPGraphQL\Type\PostObject\Connection\PostObjectConnectionResolver::sanitize_input_fields( $mock_args, null, [], $app_context, $app_info );
+
+		/**
+		 * Make sure the returned values are equal to mock args
+		 */
+		$this->assertEquals( 'testSearchString', $actual['s'] );
+		$this->assertEquals( 1, $actual['p'] );
+
+		/**
+		 * Make sure the query didn't return these array values
+		 */
+		$this->assertArrayNotHasKey( 'search', $actual );
+		$this->assertArrayNotHasKey( 'id', $actual );
+	}
+
+	public function testSanitizeInputFieldsParentArgs() {
+		$mock_args = [
+			'parent' => 2,
+			'parentIn' => [3, 4, 5],
+			'parentNotIn' => [6, 7, 8],
+			'in' => [9, 10, 11],
+			'notIn' => [12, 13, 14],
+			'nameIn' => ['testPost1', 'testPost2', 'testPost3'],
+		];
+
+		$app_context = new \WPGraphQL\AppContext();
```

Comment:
Not super essential, but to keep things a bit more DRY, instead of declaring `$app_context` and `$app_info` inside every single test, you can declare them once as a class var and just re-use them throughout the tests

For example, we setup some other re-usable values here: https://github.com/davidvexel/wp-graphql/blob/1fe1c837c83f7cfff157c8a15bc2390ccf220610/tests/test-post-connection-queries.php#L12-L34